lib/commons/aria/is-explicit-label.js

```diff
@@ -0,0 +1,34 @@
+/* global aria, dom, text */
+
+/**
+ * Note:
+ * This method is well tested via the check `explicit-label`
+ * See - `test/checks/label/explicit.js`
+ */
+
+/**
+ * Check if given element has `explicit` label
+ * @method isExplicitLabel
+ * @memberof axe.commons.aria
+ * @param {HTMLElement} node element
+ * @return {Boolean}
+ */
+aria.isExplicitLabel = function(node) {
```

Comment:
I don't think you should pull this into a commons. See my comment about using `text.textLabel`.